MinimedKit/GlucoseEvents/SensorCalGlucoseEvent.swift

```diff
@@ -11,10 +11,10 @@ import Foundation
 public struct SensorCalGlucoseEvent: RelativeTimestampedGlucoseEvent {
     public let length: Int
     public let rawData: Data
-    public let waiting: String
-    public var timestamp: DateComponents
+    public let calibrationType: String
```

Comment:
If this value is useful to public clients, its options should be defined as an enum. If it's just for debugging in `dictionaryRepresentation`, then it should be made `private`.

This comment also applies similar approaches below: `errorType`, `packetType`, `statusType`